python/imzml_reader.py

```diff
@@ -144,12 +158,20 @@ def write_zarr(self, path, dtype, compressor=None):
         help="Corresponding ibd file from Jeff Spraggins' lab",
     )
     parser.add_argument(
-        "--ims_file",
+        "--ims_zarr",
         required=True,
         help="Write the IMS data to this zarr file.",
     )
+    parser.add_argument(
+        "--ims_file",
+        type=argparse.FileType("x"),
```

Comment:
I'm having trouble finding the documentation now, but this fails if the file already exists, or something like that?

Reply:
Should we use it more broadly?

Reply:
Yup: https://docs.python.org/3/library/functions.html#open